tests/conftest.py

```diff
@@ -0,0 +1,34 @@
+import pytest
+from painless_sqlalchemy.BaseModel import Base, engine
+from painless_sqlalchemy.ModelSerialization import ModelSerialization
+
+
+@pytest.fixture(scope='session')
+def Teacher():
+    # noinspection PyShadowingNames
+    class Teacher(ModelSerialization):
+        __tablename__ = 'teacher'
+    return Teacher
+
+
+@pytest.fixture(scope='session')
+def Student():
+    # noinspection PyShadowingNames
+    class Student(ModelSerialization):
+        __tablename__ = 'student'
+    return Student
+
+
+@pytest.fixture(scope='session', autouse=True)
+def _test_config(Teacher, Student):
```

Comment:
Calling `Teacher, Student` in the signature ensures those fixtures are instantiated before this one